element_array_ephys/spike_sorting/ephys_curation.py

```diff
@@ -44,7 +44,9 @@ def activate(
         add_objects={**ephys.__dict__, **ephys_sorter.__dict__},
     )
     try:
-        ephys.ClusterQualityLabel.insert1(("unsorted", "unsorted spikes (Phy default)"), skip_duplicates=True)
+        ephys.ClusterQualityLabel.insert1(
```

Comment:
Instead of using the "unsorted" label, I think we should use "no kilosort label", so no one gets confused thinking the merged/split units still need to be sorted by the pipeline.